examples/collada.html

```diff
@@ -0,0 +1,56 @@
+<html>
+    <head>
+        <title>Itowns - collada</title>
+
+        <style type="text/css">
+            html {
+                height: 100%;
+            }
+
+            body {
+                margin: 0;
+                overflow:hidden;
+                height:100%;
+            }
+
+            #viewerDiv {
+                margin : auto auto;
+                width: 100%;
+                height: 100%;
+                padding: 0;
+            }
+
+            #menuDiv {
+                position: absolute;
+                top:0px;
+                margin-left: 0px;
+            }
+
+            @media (max-width: 600px) {
+                #menuDiv {
+                    display: none;
+                }
+            }
+        </style>
+        <meta charset="UTF-8">
+
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <script src="GUI/dat.gui/dat.gui.min.js"></script>
+    </head>
+    <body>
+        <div id="viewerDiv"></div>
+        <script src="GUI/GuiTools.js"></script>
+        <script src="../dist/itowns.js"></script>
+        <script src="../dist/debug.js"></script>
+        <script type="text/javascript">
+            var renderer;
+            var exports = {};
+            var THREE = itowns.THREE;
+            var script = document.createElement('script');
+            script.type = 'text/javascript';
+            script.src = 'https://cdn.rawgit.com/mrdoob/three.js/r' + THREE.REVISION + '/examples/js/loaders/ColladaLoader.js';
+            document.body.appendChild(script);
+        </script>
+        <script src="collada.js"></script>
```

Comment:
Actually, as I'm testing with slow network, I had cases where collada failed because ColladaLoader constructor was not defined. In fact, you also need a `defer` attribute on this `script` tag, because newer versions of firefox apparently treats dynamically inserted script as `async`. `defer` ensures they are executed after, see https://stackoverflow.com/a/8996894